### ISSUE TITLE
Error running sigploit.py with python 2.7.1 and Java 1.8

### ISSUE THREAD

**Issue by RiponDas** [SigPloiter/SigPloit#216]
root@kali:/home/kali/Desktop/SigPloit# python sigploit.py
Traceback (most recent call last):
  File "sigploit.py", line 16, in <module>
    from ss7.tracking import *
  File "/home/kali/Desktop/SigPloit/ss7/tracking.py", line 14, in <module>
    import sigploit
  File "/home/kali/Desktop/SigPloit/sigploit.py", line 21, in <module>
    from gtpmain import *
  File "/home/kali/Desktop/SigPloit/gtpmain.py", line 12, in <module>
    import gtp.info
  File "/home/kali/Desktop/SigPloit/gtp/info.py", line 18, in <module>
    from attacks.info import discover_gtp_nodes
  File "/home/kali/Desktop/SigPloit/gtp/attacks/info/discover_gtp_nodes.py", line 39, in <module>
    from gtp_v2_core.utilities.configuration_parser import parseConfigs
  File "/home/kali/Desktop/SigPloit/gtp/gtp_v2_core/utilities/configuration_parser.py", line 38, in <module>
    from configobj import ConfigObj, ConfigObjError
ImportError: No module named configobj

**Comment by Kalvelign** [comment]
use python 3.6

**Comment by Curmens** [comment]
Code is written in python2. Running with python 3 outputs an error

**Comment by RiponDas** [comment]
Both of python 2 and 3 show error, python 3 show more error than python version 2.
There is any solution please?

**Comment by Kalvelign** [comment]
just download all the depencese

Envoyé depuis ProtonMail mobile

-------- Message d'origine --------

**Comment by RiponDas** [comment]
What do you mean by "Envoyé depuis ProtonMail mobile"?
Please write details.

**Comment by Kalvelign** [comment]
envoyé depuis ProtonMail mobile mean i send this email with the app protonmail mobile from my phone

Envoyé depuis ProtonMail mobile

-------- Message d'origine --------

**Comment by TheCrazzXz** [comment]
Hey, i've got the solution for the problem :

```
https://github.com/TheCrazzXz/Get-pip 
# apt install git 
cd Get-pip
sudo python2 get-pip.py
#Check if pip2 is installed : 
pip2 --version 
pip install --upgrade setuptools
sudo apt-get install lksctp-tools
cd ..
#(Then cd to the sigploit directory)
sudo pip2 install -r requirements.txt

#(Then normally the problem is solved)
python2 sigploit.py
```

**Comment by aNDOhACK** [comment]
I Have same problem guys any help...
raceback (most recent call last):
  File "sigploit.py", line 16, in <module>
    from ss7.tracking import *
  File "/home/******/Downloads/SigPloit-master/ss7/tracking.py", line 14, in <module>
    import sigploit
  File "/home/******/Downloads/SigPloit-master/sigploit.py", line 21, in <module>
    from gtpmain import *
  File "/home/******/Downloads/SigPloit-master/gtpmain.py", line 12, in <module>
    import gtp.info
  File "/home/******/Downloads/SigPloit-master/gtp/info.py", line 18, in <module>
    from attacks.info import discover_gtp_nodes
  File "/home/******/Downloads/SigPloit-master/gtp/attacks/info/discover_gtp_nodes.py", line 39, in <module>
    from gtp_v2_core.utilities.configuration_parser import parseConfigs
  File "/home/******/Downloads/SigPloit-master/gtp/gtp_v2_core/utilities/configuration_parser.py", line 38, in <module>
    from configobj import ConfigObj, ConfigObjError
ImportError: No module named configobj

**Comment by zeroplus4** [comment]
Chat me on whatsapp for assistance
+12028885222

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by Persuasion89** [comment]
try a dif distro. I had problems on kali but it works fine on CentOS

**Comment by partalian** [comment]
It is working. Do as TheCrazzXz said, it is working.
Before I leave, just to prevent others from losing money read below:
Just paid yesterday 17/05 at 13:56 to ss7.dev $1000 (got all the proof), nothing happened.
No answer, sent 3-4 emails to them and nobody came back.
In case you stumble upon this website, beware IT IS A SCAM.
If you need ss7 then look elsewhere (don't know yet either).
ss7.dev - SCAM PAGE and money lost!

**Comment by TheCrazzXz** [comment]
@partalian Yes, and, i'm pretty much sure you didn't buy the usb device you have to use to intercept s77...
I heard that it costs about 150$ something like that...
Yeah, nothing is free ! Unfortunaly it's true
Well, sorry to disapoint y'all but sigslot ain't gonna work unless you have the usb external ss7 interceptor... 
(I don't have it)

But yea, instead of intercepting ss7 messages (and being a Script Kiddie), try to learn real hacking, try to learn programming languages (other than python) like C, C++ and even Assembly (yes Assembly is harder than programming languages and reaaaaly low level, but it's nice to understand how your program with binary instructions (translated in Assembly language), in fact it's not so hard, i don't know, if i can give you some exploits (plese note that i ain't pretending i'm a "hacker" and i ain't)
If you want some (known) exploits you can learn Stack Buffer Overflow, Buffer Overruns... Welp that's not so hard, but you should learn programming at low or very low level, please trust me, it's what makes you a real "hacker" worthy of the name 
Learn binary, learn how to memory, the processor works in your computer and who knows, maybe someone like you will find system vulnerabilities, note that the term of "hacking" means to change the behaviour of a system to make it do what we want to

Please stop learning using Kali linux softwares, or at least, you can learn it at the beggining, the beggining where everyone is a noob, because it's not by learning to use their command line that you will become a "hacker" and find vulnerabilities on a system, by using these softwares, you just exploits some of the insecured things that exists unless you add the security, or at most you will exploit some known vulnerabilities, please consider making your own programs (you should use low level languages like C or C++) like trojans, cryptography (which is important in cyber-sec), and it's justly how you will be able to bypass anti-viruses, not by using metasploit or veil-evasion or even thefatrat or else, no create your own trojans and try just i don't know, you can do weird and stupid things to like scame the antivirus, there are 1M+ methods to bypass antivirus, you can bypass it simply to really advanced things... Or... You can use something else which is... Cryptography, by encrypting the binary content of an executable, the antivirus will not be able to read the binary instructions of the encrypted program, then you can decrypt it with the key (you can also encode it instead of encrypt it) 

Anyways, a long answer isn't it ?

**Comment by partalian** [comment]
Hi CazzXz,

Thanks for the long answer, I love these ones.
I used to do some programming many many years ago, I did some encrypting to bypass viruses and I recall it was something to do with the “title” or “header” of that file and binaries too.
I agree with you regarding everything you said and that one should learn to code so that the solution you can bring is one unique and independent of any existing exploits that speculate the “now” moment and one should understand beyond the obvious step by step commands.
I like researching and indeed again coding is a game changer regardless of what you want to do.
Many thanks again.

**Comment by partalian** [comment]
By the way, I will buy the scottgadgets’ device, but the question is: will you be able to tap into ss7 with that? Or is it just to “catch” close by signal from other phones?
I am looking into this matter. Is that device enough to access the ss7 network?
Thanks again.

**Comment by partalian** [comment]
> @partalian Yes, and, i'm pretty much sure you didn't buy the usb device you have to use to intercept s77...
> I heard that it costs about 150$ something like that...
> Yeah, nothing is free ! Unfortunaly it's true
> Well, sorry to disapoint y'all but sigslot ain't gonna work unless you have the usb external ss7 interceptor...
> (I don't have it)
> 
> But yea, instead of intercepting ss7 messages (and being a Script Kiddie), try to learn real hacking, try to learn programming languages (other than python) like C, C++ and even Assembly (yes Assembly is harder than programming languages and reaaaaly low level, but it's nice to understand how your program with binary instructions (translated in Assembly language), in fact it's not so hard, i don't know, if i can give you some exploits (plese note that i ain't pretending i'm a "hacker" and i ain't)
> If you want some (known) exploits you can learn Stack Buffer Overflow, Buffer Overruns... Welp that's not so hard, but you should learn programming at low or very low level, please trust me, it's what makes you a real "hacker" worthy of the name
> Learn binary, learn how to memory, the processor works in your computer and who knows, maybe someone like you will find system vulnerabilities, note that the term of "hacking" means to change the behaviour of a system to make it do what we want to
> 
> Please stop learning using Kali linux softwares, or at least, you can learn it at the beggining, the beggining where everyone is a noob, because it's not by learning to use their command line that you will become a "hacker" and find vulnerabilities on a system, by using these softwares, you just exploits some of the insecured things that exists unless you add the security, or at most you will exploit some known vulnerabilities, please consider making your own programs (you should use low level languages like C or C++) like trojans, cryptography (which is important in cyber-sec), and it's justly how you will be able to bypass anti-viruses, not by using metasploit or veil-evasion or even thefatrat or else, no create your own trojans and try just i don't know, you can do weird and stupid things to like scame the antivirus, there are 1M+ methods to bypass antivirus, you can bypass it simply to really advanced things... Or... You can use something else which is... Cryptography, by encrypting the binary content of an executable, the antivirus will not be able to read the binary instructions of the encrypted program, then you can decrypt it with the key (you can also encode it instead of encrypt it)
> 
> Anyways, a long answer isn't it ?

Hi, please read below the 2 comments I left for you. Thanks

**Comment by TheCrazzXz** [comment]
@partalian I think so, but i can't be sure of it, but i guess you should give up if it's not working with this new thing you buy...
